query/test/unit/create_long_query_lambda/test_create_long_query.py

```diff
@@ -0,0 +1,40 @@
+from unittest.mock import patch
+
+from create_long_query_lambda.app import create_long_query
+from lambdas.long_query.create_long_query import query_db_and_put_results_in_s3
+from test.unit import QueryTestCaseUsingMockAWS
+
+
+class TestCreateLongQuery(QueryTestCaseUsingMockAWS):
+    def setUp(self):
+        super().setUp()
+        self.uuid = '26f0424a-fdce-455f-ac2e-f8f5619c6eda'
+        self.query = "SELECT * FROM files limit 10;"
+        self.mock_event = {'Records': [{'messageId': 'AAAAA',
```

Comment:
I'd make these fields static members of `TestCreateLongQuery`. Then you don't need to define `setUp` and `tearDown`.

Reply:
👍